### ISSUE TITLE
[Bug]: Variable missing when running migration tool (LLAMA_HUB_URL)

### ISSUE THREAD

**Issue by daniswhoiam** [run-llama/llama_index#11541]
### Bug Description

After I migrated to the newest version of llama_index available on pip, I needed to change the code in my existing Jupyter notebook files.
Referring to [this](https://pretty-sodium-5e0.notion.site/v0-10-0-Migration-Guide-6ede431dcb8841b09ea171e7f133bd77) migration guide, I decided to run the automatic upgrade tool with
`llamaindex-cli upgrade-file <path-to-file>`

However, when I do this, I get an error (see down below).

I have created a new conda environment with the base modules and only llama_index installed additionally, running on Python 3.10, and the error still persists. Upon inspecting the source code, I did indeed not find a LLAMA_HUB_URL variable. Furthermore, it seems to be pointing to a different file than where other seemingly related URL variables are declared.

### Version

0.10.14

### Steps to Reproduce

1. Install the newest version of llama_index (0.10.14)
2. Take a python file with a rag pipeline implementation using an older version of llama_index (0.9.xx)
3. Try to migrate the code using the automatic upgrading tool

### Relevant Logs/Tracbacks

```shell
Traceback (most recent call last):
  File "/Users/daniswhoiam/miniconda3/envs/llamaindex/bin/llamaindex-cli", line 5, in <module>
    from llama_index.cli.command_line import main
  File "/Users/daniswhoiam/miniconda3/envs/llamaindex/lib/python3.10/site-packages/llama_index/cli/command_line.py", line 7, in <module>
    from llama_index.core.llama_dataset.download import (
ImportError: cannot import name 'LLAMA_HUB_URL' from 'llama_index.core.llama_dataset.download' (/Users/daniswhoiam/miniconda3/envs/llamaindex/lib/python3.10/site-packages/llama_index/core/llama_dataset/download.py)
```

**Comment by logan-markewich** [comment]
Please try running `pip install -U llama-index-cli` -- just merged a fix

**Comment by kolaente** [comment]
I have the same problem with 0.10.29 which seems to contain the fix from #11544

**Comment by kolaente** [comment]
Well seems like I didn't understand how the llama-index core package and the cli are related. Manually upgrading the cli fixed it for me. Sorry for the noise.
